docs/docsite/rst/dev_guide/testing/sanity/ignores.rst

```diff
@@ -71,7 +71,9 @@ This will be the name of the sanity test.
 If the sanity test is specific to a version of Python, the name will include a dash (``-``) and the relevant Python version.
 If the named test uses error codes then the error code to ignore must be appended to the name of the test, separated by a colon (``:``).
 
-Below are some example ignore entries for an Ansible collection::
+Below are some example ignore entries for an Ansible collection:
+
+.. code-block::
```

Comment:
```suggestion
.. code-block:: text
```

Reply:
I was not sure about the type of the lexer to use to I went with the empty one...
thank you for you suggestion